### PR TITLE
Fix: Funnel drop off user list title is confusing (/buggy?)

### LINKED ISSUE #5218: Funnel drop off user list title is confusing (/buggy?)
## Bug description

1 - create a funnel
Screenshot 2021-07-20 at 10 39 24

2 - click '6 (54.55%)' (ie the *drop off*)

Screenshot 2021-07-20 at 10 40 52

3. Title is "Persons who completed Step #-2 - "Discovered Learning".

Yes, they completed it but this implies they succeeded.

## Expected behavior

Title to say "Persons who dropped off at Step #-2 - "Discovered Learning".

### PR DESCRIPTION
**Pull request by Patil2099**
Fixes #5218

### CODE REVIEW

File: frontend/src/scenes/funnels/funnelLogic.ts [hunk 1]
```diff
@@ -480,7 +480,7 @@ export const funnelLogic = kea<funnelLogicType>({
             personsModalLogic.actions.loadPeople({
                 action: { id: step.action_id, name: step.name, properties: [], type: step.type },
                 breakdown_value: breakdown_value || '',
-                label: `Persons who completed Step #${stepNumber} - "${step.name}"`,
+                label: `Persons who dropped off at Step #${stepNumber} - "${step.name}"`,
```
Comment (timgl) [inline frontend/src/scenes/funnels/funnelLogic.ts:483]: Thanks for creating this PR! The other issue with this was that stepNumber was a negative number, so maybe replace that with `${Math.abs(stepNumber)}`?

Comment (neilkakkar) [inline frontend/src/scenes/funnels/funnelLogic.ts:483]: Hey @Patil2099 - you'd also need to check what `stepNumber` is. If it's positive, completed is correct. If it's negative, dropping off is correct.

**Review by neilkakkar — CHANGES_REQUESTED** [review]

File: frontend/src/scenes/funnels/funnelLogic.ts (hunk 1 above)
Comment (Patil2099) [reply frontend/src/scenes/funnels/funnelLogic.ts]: Sure

Comment (Patil2099) [reply frontend/src/scenes/funnels/funnelLogic.ts]: I will make the changes

Comment (timgl) [reply frontend/src/scenes/funnels/funnelLogic.ts]: @patil2099 sorry my comment was incorrect, Neil's comment is correct!

Comment (neilkakkar) [reply frontend/src/scenes/funnels/funnelLogic.ts]: "Persons who **completed** Step X" and "Persons who **dropped off at** Step Y" , yes :)

Comment (neilkakkar) [reply frontend/src/scenes/funnels/funnelLogic.ts]: (and then, also `Maths.abs` on the number :) )

Comment (neilkakkar) [reply frontend/src/scenes/funnels/funnelLogic.ts]: Hey, just unresolved this, have a look at the two comments here^^

Comment (Patil2099) [reply frontend/src/scenes/funnels/funnelLogic.ts]: `Persons who ${stepNumber >= 0? 'completed': 'dropped off'} at Step #${stepNumber} - "${step.name}"`,

Comment (Patil2099) [reply frontend/src/scenes/funnels/funnelLogic.ts]: Is this correct ?

File: frontend/src/scenes/funnels/funnelLogic.ts [hunk 2]
```diff
@@ -480,7 +480,7 @@ export const funnelLogic = kea<funnelLogicType>({
             personsModalLogic.actions.loadPeople({
                 action: { id: step.action_id, name: step.name, properties: [], type: step.type },
                 breakdown_value: breakdown_value || '',
-                label: `Persons who completed Step #${stepNumber} - "${step.name}"`,
+                label: `Persons who ${stepNumber >= 0? 'completed': 'dropped off'} at Step #${stepNumber} - "${step.name}"`,
```
Comment (neilkakkar) [inline frontend/src/scenes/funnels/funnelLogic.ts:483]:
```suggestion
                label: `Persons who ${stepNumber >= 0? 'completed': 'dropped off at'} Step #${Math.abs(stepNumber)} - "${step.name}"`,
```

**Review by paolodamico — APPROVED** [review]
thanks for the contribution @Patil2099!

**Pull request merged**